deployment/helm/node-feature-discovery/templates/prometheus.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.prometheus.enable }}
+# Prometheus Monitor Service (Metrics)
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: {{ include "node-feature-discovery.fullname" . }}
+  labels:
+    {{- include "node-feature-discovery.selectorLabels" . | nindent 4 }}
```

Comment:
It would be helpful if i could attach a custom label to this pod monitor so specific prometheus instances can find it.